Makefile

```diff
@@ -208,7 +208,7 @@ dist: clean
 	CGO_ENABLED=0 $(GOX) -osarch="linux/amd64 linux/arm64 linux/arm darwin/amd64 windows/amd64 freebsd/amd64" ./cmd/loki-canary
 	CGO_ENABLED=0 $(GOX) -osarch="linux/arm64 linux/arm darwin/amd64 windows/amd64 freebsd/amd64" ./cmd/promtail
 	CGO_ENABLED=1 $(CGO_GOX) -osarch="linux/amd64" ./cmd/promtail
-	gzip dist/*
+	for i in dist/*; do zip $$i.zip $$i && rm -f $$i; done
```

Comment:
This actually includes dist/ prefix in ZIP archive.

Reply:
Add `-j` to fix, refer to https://stackoverflow.com/questions/9710141/create-zip-ignore-directory-structure